apps/passport-server/src/services/devconnect/organizerSync.ts

```diff
@@ -787,9 +835,41 @@ export class OrganizerSync {
           await softDeleteDevconnectPretixTicket(this.db, removedTicket);
         }
 
+        if (this.enableRedaction) {
+          // Step 4 of saving: save redacted tickets
+          for (const redactedTicket of redactedTickets) {
+            await insertDevconnectPretixRedactedTicket(this.db, {
+              hashed_email: await getHash(redactedTicket.email),
+              position_id: redactedTicket.position_id,
```

Comment:
```
Indexes:
    "devconnect_pretix_tickets_position_id_key" UNIQUE CONSTRAINT, btree (position_id)
```

We already assume that it is unique for tickets. We could make it unique per organizer, perhaps, since organizers always live on the same server and presumably the position ID is unique within that namespace.